### ISSUE TITLE
No UI showing up

### ISSUE THREAD

**Issue by dan-m8t** [wwmm/easyeffects#2178]
Hi,
I reinstalled my system and wanted to install easyeffects also.

I'm running on Arch. 

Easyeffects does not start or better, it does not show any UI.

I installed from flatpak, Arch community repo and aur (easyeffects-git) - all with the same behaviour.

At first I got an `Failed to register` timeout message, that is somehow gone, not sure why.

When I run `G_MESSAGES_DEBUG=easyeffects easyeffects` everythings looks fine though, no more errors, but no UI also.

Here's the debug output:

```G_MESSAGES_DEBUG=easyeffects easyeffects                                                                                                                                                                                       KILL ✘  3m 36s  
(process:52067): easyeffects-DEBUG: 23:08:52.188: 	easyeffects.cpp:35	easyeffects version: 7.0.1
(process:52067): easyeffects-DEBUG: 23:08:52.189: 	easyeffects.cpp:45	locale directory: /usr/share/locale
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:49	system input presets directory: "/etc/easyeffects/input"; 
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:49	system input presets directory: "/etc/xdg/easyeffects/input"; 
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:52	system output presets directory: "/etc/easyeffects/output"; 
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:52	system output presets directory: "/etc/xdg/easyeffects/output"; 
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:194	user presets directory already exists: /home/dan/.config/easyeffects/
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:194	user presets directory already exists: /home/dan/.config/easyeffects/input
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:194	user presets directory already exists: /home/dan/.config/easyeffects/output
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:194	user presets directory already exists: /home/dan/.config/easyeffects/autoload/input
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.190: 	presets_manager.cpp:194	user presets directory already exists: /home/dan/.config/easyeffects/autoload/output
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.239: 	pipe_manager.cpp:1426	compiled with PipeWire: 0.3.67
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.239: 	pipe_manager.cpp:1427	linked to PipeWire: 0.3.67
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.244: 	pipe_manager.cpp:1397	core version: 0.3.67
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.244: 	pipe_manager.cpp:1398	core name: pipewire-0
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.244: 	pipe_manager.cpp:1311	found metadata: settings
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.244: 	pipe_manager.cpp:1311	found metadata: default
(easyeffects:52067): easyeffects-DEBUG: 23:08:52.244: 	pipe_manager.cpp:1311	found metadata: route-settings
```

I can see with `ps aux` that easyeffects is running in the background.
When I start easyeffects with the GNOME launcher it is also running in the background according to `ps aux`:

`dan        52285  2.6  0.1 553848 42720 ?        Sl   23:10   0:00 /usr/bin/easyeffects --gapplication-service`

Still no UI.
I'm using GNOME 43.3 on Wayland

Any hint?

Thanks

**Comment by wwmm** [comment]
>I installed from flatpak, Arch community repo and aur (easyeffects-git) - all with the same behaviour.

Strange. I also use Arch Linux + GNOME + Wayland. Nothing happened recently that could explain the window not being shown.

>Here's the debug output:

Is this the full log? Under normal circumstances there should be more lines after `found metadata: route-settings`. 

>At first I got an Failed to register timeout message, that is somehow gone, not sure why.

Hum... Maybe some problem in dbus or gtk4. Not a good sign indeed.

**Comment by dan-m8t** [comment]
Yes this is the full log, nothing even happens after that, cannot even CTRL + C out of it, need to send SIGTERM.


Maybe as a hint to gtk4:

I wanted to use KDE at first, but Wayland support in KDE is not that good, so I went back to old GNOME. I uninstalled KDE of course, but I am not sure if it left some traces to gtk4 ?! But to be clear, easyeffects is the only application that won't run at the moment, I tried other gtk4 apps like Clapper without a problem.
Not sure where to look further though :(.
